lib/uri.js

```diff
@@ -110,7 +110,7 @@ URI.isValid = function(arg, knownParams) {
 URI.parse = function(uri) {
   var info = URL.parse(uri, true);
 
-  if (info.protocol !== 'bitcoincash:') {
+  if (info.protocol !== 'bitcoincash:' && info.protocol !== 'bchtest:') {
```

Comment:
May be it would be better to use:
```
if (Networks.get( info.protocol ,'prefix')) {
```

(see)
https://github.com/bitpay/bitcore-lib-cash/blob/master/lib/networks.js#L193